releases.md

```diff
@@ -36,6 +36,13 @@ Further documentation available:
 
 ## Releases
 
+### v0.28
+
+- **Latest Release**: [v0.28.0][v0-28-0] (2022-11-25) ([docs][v0-28-0-docs])
+- **Initial Release**: [v0.28.0][v0-28-0] (2022-11-25)
+- **End of Life**: 2023-02-25
```

Comment:
@tektoncd/cli-maintainers what should be the EOL for this release? Should it be 3 months or more?